Examples/example_esp_async.py

```diff
@@ -0,0 +1,43 @@
+# The MIT License (MIT)
+# Copyright (c) 2020 Mike Teachman
+# https://opensource.org/licenses/MIT
+
+# example for MicroPython rotary encoder
+#
+# Documentation:
+#   https://github.com/MikeTeachman/micropython-rotary
+
+from rotary_irq_esp import RotaryIRQ
```

Comment:
this example will also work for Pyboards.  I suggest to change the filename to "example_asyncio.py" and 

```
import sys
if sys.platform == 'esp8266' or sys.platform == 'esp32':
    from rotary_irq_esp import RotaryIRQ
elif sys.platform == 'pyboard':
    from rotary_irq_pyb import RotaryIRQ
else:
    print('Warning:  The Rotary module has not been tested on this platform')  
```

Reply:
ok, sounds good to me.